src/components/AcyPoolComponent/index.js

```diff
@@ -1,104 +1,482 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import useSWR from 'swr';
+import { ethers } from 'ethers';
+import { Tooltip } from 'antd'
+import { useChainId } from '@/utils/helpers';
+import { useWeb3React } from '@web3-react/core';
 import { useConstantLoader } from '@/constants';
-import { GlpSwapBox, GlpSwapDetailBox } from '../PerpetualComponent/components/GlpSwapBox';
-import Reader from '@/acy-dex-futures/abis/Reader.json'
-import GlpManager from '@/acy-dex-futures/abis/GlpManager.json'
-import Glp from '@/acy-dex-futures/abis/ERC20.json'
-import useSWR from 'swr'
+import { getTokens, getContract } from '@/constants/future_option_power.js';
+import { useConnectWallet } from '@/components/ConnectWallet';
+import { approveTokens, addLiquidity, removeLiquidity } from '@/services/derivatives';
 import {
-  USD_DECIMALS,
-  GLP_DECIMALS,
-  PLACEHOLDER_ACCOUNT,
-  expandDecimals,
-  fetcher,
-  getInfoTokens,
   bigNumberify,
-} from '@/acy-dex-futures/utils'
-import { useChainId } from '@/utils/helpers';
-import { getTokens, getContract } from '@/constants/powers.js';
+  parseValue,
+  fetcher,
+  expandDecimals,
+  formatAmount,
+  formatAmountFree,
+  PLACEHOLDER_ACCOUNT,
+  ALP_DECIMALS,
+  USD_DECIMALS,
+  LONGDIGIT,
+  SHORTDIGIT,
+  BASIS_POINTS_DIVISOR,
+  MINT_BURN_FEE_BASIS_POINTS,
+  BURN_FEE_BASIS_POINTS,
+} from '@/acy-dex-futures/utils';
+import PerpetualTabs from '@/components/PerpetualComponent/components/PerpetualTabs';
+import BuyInputSection from '@/pages/BuyGlp/components/BuyInputSection';
+import glp40Icon from '@/pages/BuyGlp/components/ic_glp_40.svg'
+import AcyPerpetualButton from '@/components/PerpetualComponent/components/AcyPerpetualButton';
+import ERC20 from '@/acy-dex-futures/abis/ERC20.json';
+import Token from '@/acy-dex-futures/abis/Token.json';
+import Router from '@/abis/future-option-power/Router.json';
+import Reader from '@/abis/future-option-power/Reader.json';
+import Alp from '@/abis/future-option-power/Alp.json'
```

Comment:
In current smart contract, we use [`MintableToken`](https://github.com/ACY-Labs/acy-futures-options-powers/blob/develop_liquidityV2/contracts/pool/PoolImplementation.sol#L382) as template to create LP token. Thus we should use [`MintableToken`](https://github.com/ACY-Labs/acy-futures-options-powers/blob/develop_liquidityV2/contracts/token/MintableToken.sol) as LP token's ABI.

Reply:
Joy update: `Alp.json` is actually `MintableToken`'s abi format.